pkg/formatted/k8s.go

```diff
@@ -29,11 +31,25 @@ var ConditionColor = map[string]color.Attribute{
 	"Pending":   color.FgHiWhite,
 }
 
+var stepCounter = -1
+
 // ColorStatus Get a status coloured
 func ColorStatus(status string) string {
 	return color.New(ConditionColor[status]).Sprint(status)
 }
 
+// AutoStepName when our stepName is empty return a genrated name as generated
+// on pipeLine
+func AutoStepName(stepName string) string {
+	if stepName != "" {
+		return stepName
+	}
+	// I don't think we should care about rwlock, gothread and stuff like this
+	// since it's only use in describe
+	stepCounter++
```

Comment:
agree although `atomic.AddUint64(&stepCounter, 1)` does the same without the need for locks

Reply:
sound easy enough...